### DISCUSSION TITLE
https://tieba.baidu.com/mg/o/getForumHome 这个接口还能用吗

### DISCUSSION THREAD

**Discussion by leezee7** [MoeNetwork/Tieba-Cloud-Sign#205] [疑问] [answered]
谷歌浏览器登录了百度帐号，然后访问 https://tieba.baidu.com/mg/o/getForumHome?st=0&pn=1&rn=200，请求是携带了 Cookie 的，但是没有返回关注列表。

刚通过 docker 安装的最新版本。

**Comment by BANKA2017** [comment, marked as answer]
补充完整下面代码执行一下看看有什么结果，敏感信息可以打码

```bash
curl 'https://tieba.baidu.com/mg/o/getForumHome?st=0&pn=1&rn=200' -H 'Cookie: BDUSS=填上bduss; STOKEN=填上贴吧stoken;' --compressed
```

类似的问题在 #186 有人提出但我无法复现

**Reply by leezee7** [reply]
> 补充完整下面代码执行一下看看有什么结果，敏感信息可以打码
> 
> ```shell
> curl 'https://tieba.baidu.com/mg/o/getForumHome?st=0&pn=1&rn=200' -H 'Cookie: BDUSS=填上bduss; STOKEN=填上贴吧stoken;' --compressed
> ```
> 
> 类似的问题在 #186 有人提出但我无法复现

```json
{
    "data": {
        "recomm_forum": [
            {
                "avatar": "https://imgsa.baidu.com/forum/pic/item/b03533fa828ba61e6434203a4934970a314e59c9.jpg",
                "forum_id": 122873,
                "forum_name": "死神",
                "hot_num": 22661,
                "info": null,
                "link": "/f?ie=utf-8&kw=死神&fr=search"
            }
        ],
        "tokens": {
            "bottom_banner": "$pwn@@$",
            "bottom_layer": "$pwf@@$",
            "index_footer_client_down": "$pwk@@$",
            "index_message_icon": "$pwt@@$"
        },
        "ubs_abtest_config": null,
        "ubs_sample_ids": null,
        "user_info": {
            "id": 0,
            "is_login": 0
        }
    },
    "errmsg": "成功",
    "errno": 0,
    "logid": "1375495373",
    "server_time": 43239,
    "time": "2022-09-12 20:22:55.542110386 +0800 CST m=+26929.690760794"
}
```

看着像 BDUSS/STOKEN 有错误，但是又能绑定成功，也能显示出贴吧帐号昵称。

**Reply by BANKA2017** [reply]
嗯，出现这个看起来是stoken有问题，能绑定成功是因为获取那些信息是不需要stoken的

**Reply by leezee7** [reply]
![image](https://user-images.githubusercontent.com/24501682/189659582-c2fb63b7-84fc-476c-8f30-44792b0cd36f.png)


请问是复制这个吗？

**Reply by BANKA2017** [reply]
> ![image](https://user-images.githubusercontent.com/24501682/189659582-c2fb63b7-84fc-476c-8f30-44792b0cd36f.png)
> 
> 请问是复制这个吗？

不是的，找域名是 `.tieba.baidu.com` 的那个
![image](https://user-images.githubusercontent.com/17263253/189668887-5ecd56ad-10ba-43d4-9bc6-5483a4925e19.png)

**Reply by leezee7** [reply]
可以了，感谢🙏🏻
